executable/microsoft_pe.ksy

```diff
@@ -408,3 +413,144 @@ types:
         size: length - 8
         doc: Contains a certificate, such as an Authenticode signature.
     doc-ref: 'https://docs.microsoft.com/en-us/windows/desktop/debug/pe-format#the-attribute-certificate-table-image-only'
+
+  resource_directory_table:
+    params:
+      - id: depth
+        type: u4
+      - id: rsrc_section_file_offset
+        type: u4
+      - id: rsrc_section_virtual_address
+        type: u4
+    seq:
+      - id: characteristics
+        type: u4
+      - id: time_date_stamp
+        type: u4
+      - id: major_version
```

Comment:
I'd like to move version into an own type

Reply:
I was thinking maybe it would become a bit too verbose? The rest of the file doesn't use specific type for versions in structure. Maybe keeping this as-is to be consistent with the rest of the file?

Reply:
IMHO: code self-documentation and semantic structuring is almost always beneficial.

Reply:
aight, I've created `resource_directory_table_version`, please take a look